src/main/resources/application.yaml

```diff
@@ -96,6 +96,7 @@ scheduling:
       ttl: PT72H # ISO-8601
       cron: 0 0 6 * * *
     check-new-envelopes:
+      time-interval: 60 # In minutes
```

Comment:
did you consider using `PT60M` instead? We use iso-8601 in link 96 and 83

Reply:
I tried `PT60M` but when trying to get the value, `timeInterval.get(ChronoUnit.MINUTES)`  returns ` java.time.temporal.UnsupportedTemporalTypeException: Unsupported unit: Minutes`.

Reply:
`Duration.get()` supports only a few types. I've updated to use `seconds`.